### PR TITLE
Update `Guides/Tablet Purchase`

### PR DESCRIPTION
**Pull request by wowcake**
Added some missing word, and picture, also bringing some other tablet models that are often used for osu!.

### CODE REVIEW

File: wiki/Guides/Tablet_Purchase/en.md [hunk 1]
```diff
@@ -48,22 +48,37 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
 
 ![](https://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
 
-![](https://puu.sh/dlQi "Third generation Bamboo (Connect, Capture, Create - Connect shown)")
+![](https://wowcake.s-ul.eu/PgLyCagy "Third generation Bamboo (Connect, Capture, Create - Connect shown)")
```
Comment (peppy) [inline wiki/Guides/Tablet_Purchase/en.md:51]: image should be included in PR. we don't support external hosting like this.

File: wiki/Guides/Tablet_Purchase/en.md [hunk 2]
```diff
@@ -48,22 +48,37 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
+
+Although, some people still prefer the Wacom's tablets, despite of those alternatives.
+
+In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used CTL-480 if you're also into art.
 
 ## Recommendations
 
 Here are some links to purchase the latest models.
 
+- [Amazon - One by Wacom Tablet (small version) (CTL472)](https://www.amazon.com/dp/B07S1RR3FR/ref=cm_sw_r_awdo_navT_g_6QR3RY32BE2C0WQ8B1KG)
```
Comment (peppy) [inline wiki/Guides/Tablet_Purchase/en.md:81]: I hope this isn't intentional malice, but you are attempting to include a referral code in this link.

**Review by peppy — CHANGES_REQUESTED** [review]

File: wiki/Guides/Tablet_Purchase/en.md (hunk 1 above)
Comment (wowcake) [reply wiki/Guides/Tablet_Purchase/en.md:51]: Right, i'll change it to use the images that are already provided here.

File: wiki/Guides/Tablet_Purchase/en.md (hunk 2 above)
Comment (wowcake) [reply wiki/Guides/Tablet_Purchase/en.md:81]: my bad, didn't realize that the link includes referral code

File: wiki/Guides/Tablet_Purchase/en.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ With the goal of trying to allow players to try the tablet input method without
...
+Huion is a tablet company that is the supplier of the osu!tablet. They are best known for being cheaper than Wacom tablets by a large margin but some would say that their tablets feel flimsier and that running a tablet pen over a Huion tablet would feel as if you were on writing paper (in comparison, some say that on Wacom, it is as if you were writing on glass). Their tablet pens either require a battery or needs to be charged, unlike Wacom tablet pens, and their nibs may need to be often replaced than Wacom nibs. Other than that, there really are not any noticeable differences between Walcom and Huion tablets.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:29]:
```suggestion
Huion is a tablet company that is the supplier of the osu!tablet. They are best known for being cheaper than Wacom tablets by a large margin but some would say that their tablets feel flimsier and that running a tablet pen over a Huion tablet would feel as if you were on writing paper (in comparison, some say that on Wacom, it is as if you were writing on glass). Their tablet pens either require a battery or needs to be charged, unlike Wacom tablet pens, and their nibs may need to be replaced more often than Wacom nibs. Other than that, there really are not any noticeable differences between Wacom and Huion tablets.
```

File: wiki/Guides/Tablet_Purchase/en.md [hunk 4]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
 
-Lastly, Intuos3 tablets can be had for very cheap on second-hand sites like eBay or Craigslist, and are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.
+Lastly, CTL-480 tablets can be had for very cheap on second-hand sites like eBay or Amazon, although they are often hard to find, due to the discontinuation of the manufacturing, but if you could find one for a reasonable price, they are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:59]: what about ctl-470 and other similar models?

File: wiki/Guides/Tablet_Purchase/en.md [hunk 5]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
 
-In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used Intuos3 if you're also into art.
+### Other tablet brands
+
+Other than Wacom and Huion, there are also more tablet manufacturers that tries to compete with the market, bringing a decent product that targetted at artists, and even osu! players.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:63]:
```suggestion
Beside Wacom and Huion, there are other competitors on the market, who manufacture decent tablets targeted at artists, and even osu! players.
```

File: wiki/Guides/Tablet_Purchase/en.md [hunk 6]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
+
+Some of those manufacturer and their known tablet are:
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:65]:
```suggestion
Some of those manufacturers are:
```

File: wiki/Guides/Tablet_Purchase/en.md [hunk 7]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
+
+Although, some people still prefer the Wacom's tablets, despite of those alternatives.
+
+In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used CTL-480 if you're also into art.
 
 ## Recommendations
 
 Here are some links to purchase the latest models.
 
+- [Amazon - One by Wacom Tablet (small version) (CTL472)](https://www.amazon.com/Wacom-Graphic-Drawing-Tablet-Beginners/dp/B07S1RR3FR)
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:81]: can cut it even further
```suggestion
- [Amazon - One by Wacom Tablet (small version) (CTL472)](https://www.amazon.com/dp/B07S1RR3FR)
```

File: wiki/Guides/Tablet_Purchase/en.md [hunk 8]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
-In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used Intuos3 if you're also into art.
+### Other tablet brands
+
+Other than Wacom and Huion, there are also more tablet manufacturers that tries to compete with the market, bringing a decent product that targetted at artists, and even osu! players.
+
+Some of those manufacturer and their known tablet are:
+
+- Gaomon (known for their Gaomon S620 tablet).
+- XP-Pen (known for their XP-PEN G640 Rev A tablet).
+- VEIKK  (known for their VEIKK S640 tablet).
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:69]:
```suggestion
- VEIKK (known for their VEIKK S640 tablet).
```

File: wiki/Guides/Tablet_Purchase/en.md [hunk 9]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
+
+All of those aforementioned tablets have a pretty decent quality. Often being priced much cheaper than Wacom's tablets, making them a really good tablet when you're on budget.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:71]:
```suggestion
All of tablets mentioned are of decent quality and often cost less than Wacom's tablets, which makes them good budget options.
```

File: wiki/Guides/Tablet_Purchase/en.md [hunk 10]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
+
+Although, some people still prefer the Wacom's tablets, despite of those alternatives.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:73]: this sentence sounds unnecessary

File: wiki/Guides/Tablet_Purchase/en.md [hunk 11]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
+
+In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used CTL-480 if you're also into art.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:75]: I think everything else is excessive; besides, there's a section with the recommendations right below
```suggestion
In short, Wacom is recommended by most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!.
```

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Guides/Tablet_Purchase/en.md (hunk 9 above)
Comment (Walavouchey) [reply wiki/Guides/Tablet_Purchase/en.md:71]:
```suggestion
All of the tablets mentioned are of decent quality and often cost less than Wacom's tablets, which makes them good budget options.
```

File: wiki/Guides/Tablet_Purchase/en.md (hunk 7 above)
Comment (wowcake) [reply wiki/Guides/Tablet_Purchase/en.md:81]: thanks

File: wiki/Guides/Tablet_Purchase/en.md (hunk 10 above)
Comment (wowcake) [reply wiki/Guides/Tablet_Purchase/en.md:73]: agree, removed it

File: wiki/Guides/Tablet_Purchase/en.md (hunk 11 above)
Comment (wowcake) [reply wiki/Guides/Tablet_Purchase/en.md:75]: got it

File: wiki/Guides/Tablet_Purchase/en.md [hunk 12]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
 
 ![](img/gen1.jpg "First generation Bamboo (Pen and Touch, Touch, and Fun - Bamboo Fun shown)")
 
-![](https://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
+![](img/gen2.jpg "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
```
Comment (peppy) [inline wiki/Guides/Tablet_Purchase/en.md:49]: I'm pretty sure these images are missing in the PR?

Comment (TicClick) [reply wiki/Guides/Tablet_Purchase/en.md:49]: it looks like they were there all the time (since 4 years ago, yeah)

Comment (wowcake) [reply wiki/Guides/Tablet_Purchase/en.md:49]: Appear just fine on mine and https://github.com/ppy/osu-wiki/tree/master/wiki/Guides/Tablet_Purchase/img
IMG_20210913_161654

Comment (peppy) [reply wiki/Guides/Tablet_Purchase/en.md:49]: Ah, right. Thought they were new ones.

File: wiki/Guides/Tablet_Purchase/en.md [hunk 13]
```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
...
 
-Lastly, Intuos3 tablets can be had for very cheap on second-hand sites like eBay or Craigslist, and are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.
+Lastly, CTL-480 tablets can be had for very cheap on second-hand sites like eBay or Amazon, although they are often hard to find, due to the discontinuation of the manufacturing, but if you could find one for a reasonable price, they are definitely worth checking out if you're also going to be doing digital art as well. Beside the Wacom CTL-480, that is commonly used and praised by players. There is also the CTL-470 lineup (such as the 470, 471 and 472) and CTL-4100, which is the latest models from Wacom, offering the same quality, if not better than the CTL-480.
```
Comment (TicClick) [inline wiki/Guides/Tablet_Purchase/en.md:59]:
```suggestion
Lastly, CTL-480 tablets may be sold cheap on second-hand sites like eBay or Amazon, although they are often hard to find due to the discontinuation of the manufacturing. If you could find one for a reasonable price, they are definitely worth checking out, especially if you're also a digital artist. Beside Wacom CTL-480, which is commonly used and praised by players, there is also the CTL-470 lineup (such as the 470, 471 and 472 models), and the latest CTL-4100 series, which offers the same quality, if not better than CTL-480.
```

**Review by TicClick — CHANGES_REQUESTED** [review]
I don't have a tablet, so I can only comment on the paragraph's structure

**Review by peppy — APPROVED** [review]

**Pull request merged**